### PR TITLE
Update Bridges section to be Interoperability section, with XCM and B…

### PR DESCRIPTION
**Pull request by emresurmeli**

### CODE REVIEW

File: docs/learn/learn-architecture.md [hunk 1]
```diff
@@ -55,8 +55,15 @@ those of the Relay Chain validator set and no other. Since the validator set on
 expected to be secure with a large amount of stake put up to back it, parachains should benefit from
 this security.
 
-## [Bridges](learn-bridges.md)
+## Interoperability
 
+### [XCM](learn-crosschain.md)
+XCM, short for cross-consensus messaging is a format, and not a protocol. It does not assume anything
```
Comment (salmad3) [inline docs/learn/learn-architecture.md]:
```suggestion
XCM, short for cross-consensus messaging, is a format and not a protocol. The format does not assume anything
```

File: docs/learn/learn-architecture.md [hunk 2]
```diff
@@ -55,8 +55,15 @@ those of the Relay Chain validator set and no other. Since the validator set on
...
+about the reciever or senders consensus mechanism, it just cares about the format in which the messages
```
Comment (salmad3) [inline docs/learn/learn-architecture.md]:
```suggestion
about the receiver or senders consensus mechanism, it only cares about the format in which the messages
```

File: docs/learn/learn-architecture.md [hunk 3]
```diff
@@ -55,8 +55,15 @@ those of the Relay Chain validator set and no other. Since the validator set on
...
+must be formatted in. This format is the bread and butter of how parachains can communicate with one
```
Comment (salmad3) [inline docs/learn/learn-architecture.md]:
```suggestion
must be formatted in. XCM is how parachains will be able to communicate with one
```

**Review by salmad3 — CHANGES_REQUESTED** [review]

**Comment by DrW3RK** [comment]
The article can benefit from a summary of these discussions
https://substrate.stackexchange.com/questions/151/what-is-the-xcm-format-on-polkadot
https://substrate.stackexchange.com/questions/591/what-is-the-difference-between-xcm-and-xcmp

File: docs/learn/learn-architecture.md [hunk 4]
```diff
@@ -55,8 +55,17 @@ those of the Relay Chain validator set and no other. Since the validator set on
 expected to be secure with a large amount of stake put up to back it, parachains should benefit from
 this security.
 
-## [Bridges](learn-bridges.md)
+## Interoperability
 
+### [XCM](learn-crosschain)
+XCM, short for cross-consensus message, is a format and not a protocol. The format 
+does not assume anything about the receiver or senders consensus mechanism, it only cares about 
+the format in which the messages must be structured in. XCM format is how parachains will be able to
```
Comment (salmad3) [inline docs/learn/learn-architecture.md]:
```suggestion
the format in which the messages must be structured in. The XCM format is how parachains will be able to 
```

File: docs/learn/learn-architecture.md [hunk 5]
```diff
@@ -55,8 +55,17 @@ those of the Relay Chain validator set and no other. Since the validator set on
...
 this security.
 
-## [Bridges](learn-bridges.md)
+## Interoperability
 
+### [XCM](learn-crosschain)
+XCM, short for cross-consensus message, is a format and not a protocol. The format 
+does not assume anything about the receiver or senders consensus mechanism, it only cares about 
+the format in which the messages must be structured in. XCM format is how parachains will be able to 
+communicate with one another. Different from XCMP which is short for cross-chain messaging protocol,
```
Comment (salmad3) [inline docs/learn/learn-architecture.md]:
```suggestion
communicate with one another. Different from XCMP, which is short for cross-chain messaging protocol,
```

File: docs/learn/learn-architecture.md [hunk 6]
```diff
@@ -55,8 +55,17 @@ those of the Relay Chain validator set and no other. Since the validator set on
...
+XCM is what gets delivered, and XCMP is the delivery mechanism. Best way to learn more about XCM by
```
Comment (salmad3) [inline docs/learn/learn-architecture.md]:
```suggestion
XCM is what gets delivered, and XCMP is the delivery mechanism. The best way to learn more about XCM is by
```

**Review by salmad3 — CHANGES_REQUESTED** [review]

**Review by salmad3 — APPROVED** [review]

**Pull request merged**